backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java

```diff
@@ -316,9 +316,9 @@ public CreateExperimentRun getCreateExperimentRunRequest(
                 Artifact.newBuilder()
                     .setKey("Google developer Observation artifact")
                     .setPath("This is data artifact type in Google developer Observation artifact")
-                    .setArtifactType(ArtifactType.DATA)
-                    .build())
+                    .setArtifactType(ArtifactType.DATA))
```

Comment:
This changed is not needed. but its fine if it works.

Reply:
it works and I highly recommend following it in similar changes in future .

Reply:
Ok, make sense